docs/openhabian-amanda.md

```diff
@@ -567,12 +567,22 @@ You could also move that temporary recovered image file to your Windows PC that
+
+Example files are
+/volume1/rp3Backup/amanda-backups/amanda_data_20200331020001.tar.gz
+/volume1/rp3Backup/slots/slot14/00000.openHABian-openhab-dir-014
+/volume1/rp3Backup/slots/slot14/00001.openhab._var_lib_openhab2.1
+/volume1/rp3Backup/slots/slot14/00002.openhab._etc_openhab2.0
+
+You can extract the /var/lib/amanda files from the amanda_data_DATE.tar.gz file you want (probably the latest). You can also extract the /var/log/amanda directory if needed.
 So you can reinstall openHABian including Amanda from scratch and copy back those extracted files. Eventually see `amadmin import` option.
 Even if you fail to recover your index files, you can still access the files in your storage area.
-The `amindex` command can be used to regenerate the database. How to apply unfortunately is  out of scope for this document so please g**gle if needed.
+The `amindex` command can be used to regenerate the database. How to apply unfortunately is out of scope for this document so please g**gle if needed.
+
 There's also a manual way: Amanda storage files are tar files of the destination directory or compressed raw copies of partitions, both have an additional 32KB header.
-If you just want to retrieve some files (such as the items, things, rules, sitemaps, icons and the jsondb information written directly by the paperUI or HABpanelUI) from a partition backup file in the slots directory, you can mount/download that file. See https://major.io/2010/12/14/mounting-a-raw-partition-file-made-with-dd-or-dd_rescue-in-linux/.
+If you just want to retrieve some files (such as the items, things, rules, sitemaps, icons and the jsondb information written directly by the paperUI or HABpanelUI) from a partition backup file in the slots directory, you get the relevant slot numbered files from your storage device corresponding to the date you need. The openhab._etc_openhab2.0 file contains the items, icons, sitemaps and similar folders while the openhab._var_lib_openhab2.1 file contains config, etc, jsondb and other folders. Basically you run dd plus tar tvf to list the contents and dd plus tar xvf to extract the files. You can then put them back in the correct location on your openhabian box. You do not even have to put back the amanda_data files since amanda will just build new indexes based on the new daily backups, just like it did the first time amanda was installed. Just make sure you make an img backup of all your hard work!!
```

Comment:
Sorry this still doesn't hit the point.
You can use tar to extract _directory dumps_ because they're tar files, but you can NOT use dd or tar to extract _partition dumps_. That's why you need to _mount_ those. Maybe give an example of that, too (in addition to the example which is already there but shows the tar variant only)

Reply:
> Just make sure you make an img backup of all your hard work!!

remove this statement as it's unneeded when you have Amanda back in operation (and if you don't it's unclear what it would mean)

Reply:
Sorry about _var_ switching to italics. It only shows up after I have done a push from VScode. I have left the "get the relevant files from your storage device" as device neutral. I do not use a usb device and cannot show a mount example. Perhaps you use a usb device. I think an example showing how to mount and get back the amanda_data_DATE.tar,gz file from your device would really help the community

Reply:
Try yourself, you don't need USB. Add /dev/mmcblk0 to disklist, that's your SD card.
The very last line in current README tells you how to extract it, then just mount following explanations in the URL. Did this here. Add these example lines to the end of README:
(change last existing line to match dates, filenames)

```
[12:31:38] root@openhabianpi:/volatile/backup# fdisk -l /volatile/temp/restore_file
Disk /volatile/temp/restore_file: 14,9 GiB, 15931539456 bytes, 31116288 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0xfc7531cb

Device                       Boot Start      End  Sectors  Size Id Type
/volatile/temp/restore_file1       8192    93236    85045 41,5M  c W95 FAT32 (LBA)
/volatile/temp/restore_file2      94208 14774271 14680064    7G 83 Linux
[12:32:09] root@openhabianpi:/volatile/backup# echo '94208*512' | bc
48234496
[12:34:57] root@openhabianpi:/volatile/backup# mount -o loop,offset=48234496 restore_file /mnt
mount: /mnt: failed to setup loop device for restore_file.
[12:37:32] root@openhabianpi:/volatile/backup# mount -o loop,offset=48234496 /volatile/temp/restore_file /mnt
[12:37:45] root@openhabianpi:/volatile/temp# ls -l /mnt/etc/openhab2/
insgesamt 56
drwxrwxr-x  5 openhab openhab 4096 Jan 18 17:59 automation
drwxrwxr-x  2 openhab openhab 4096 Apr 19 17:05 html
drwxrwxr-x  3 openhab openhab 4096 Jan 21 10:07 icons
drwxrwxr-x  2 openhab openhab 4096 Apr 19 17:05 items

[...]
```

Reply:
Thanks, I have a 32GB SD card so will setup a separate amanda config to do this. Your example has been added